resources/views/MyAccount/MyAccount.twig

```diff
@@ -76,8 +76,7 @@
                 <p class="text-muted small font-italic">{{ trans("Ceres::Template.myAccountChangeLoginData") }}</p>
 
                 <div class="card mb-5">
-                    <account-settings template="#vue-account-settings" :user-data="{{ userObject }}">
-                        <account-settings>
+                    <account-settings template="#vue-account-settings" :user-data="{{ userObject }}"></account-settings>
```

Comment:
da du das Template als default in der Komponente gesetzt hast, kann folgender Code entfernt werden:

```template="#vue-account-settings"```

Reply:
done